script/install-flynn.tmpl

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+#
+# A script to install Flynn from a TUF repository.
+
+set -eo pipefail
+
+FLYNN_HOST_CHECKSUM="{{FLYNN-HOST-CHECKSUM}}"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 [options]
+
+OPTIONS:
+  -h            Show this message
+  -r URL        The TUF repository to download files from [default: https://dl.flynn.io]
+USAGE
+}
+
+main() {
+  if ! is_root; then
+    fail "this script must be executed as the root user"
+  fi
+
+  check_installed "curl" "sha512sum" "apt-get" "apt-key" "initctl"
+
```

Comment:
For now we should ensure the `/etc/os-release` file says that we are running some version of Ubuntu 14.04 as well.